src/com/galactanet/gametable/events/PogMenuRenderEvent.java

```diff
@@ -0,0 +1,30 @@
+package com.galactanet.gametable.events;
+
+import com.galactanet.gametable.Pog;
+
+import javax.swing.JPopupMenu;
+
+public class PogMenuRenderEvent {
```

Comment:
I couldn't come up with a generic way of doing this that would allow access to the data needed, without the listener having to do a load of extra checking of the event and the types of properties etc. Welcome suggestions though.